proto/accessgraph/v1alpha/access_graph_service.proto

```diff
@@ -312,3 +542,191 @@ message NetIQSyncOperation {}
 
 // NetIQEventsStreamResponse is a response from NetIQEventsStream
 message NetIQEventsStreamResponse {}
+
+// AWSCloudTrailStreamRequest is sent from the client to the server over the
+// bi-directional AWSCloudTrailStream. It encapsulates distinct client actions for
+// configuring the export AWS stream, sending batches of cloud trail events, and
+// managing the persistent resume state maintained by the server.
+//
+// The message uses a `oneof` payload to represent one of three distinct actions:
+//  - Config for Synchronization
+//  - Events in bulk as raw data
+//  - Events from files as gzipped data
+//
+// Behavior and Constraints:
+// - Direction: Client -> Server only.
+// - Initial Message: First message on stream MUST be `config`.
+// - State Management:
+//   - `events` includes `resume_state` for persistent progress tracking.
+message AWSCloudTrailStreamRequest {
+  oneof action {
+    AWSCloudTrailConfig config = 1; // Export configuration, e.g. start date. // Config negatiation phase
+    AWSCloudTrailEvents events = 2; // Batch of  log events and resume state.
```

Comment:
we can drop this. We are not using it at the moment.
We had to drop it to reduce athena costs